dts/bindings/adc/ti,ads1x1x-base.yaml

```diff
@@ -0,0 +1,11 @@
+# Common fields for Texas Instruments ADS1x1x
+
+include: [adc-controller.yaml, i2c-device.yaml]
+
+properties:
+    "#io-channel-cells":
+      const: 2
```

Comment:
I think you can use an `enum` instead of `const` to only allow 1 and 2

Reply:
Differential channels are not currently supported; only single-ended is supported.

There's a longstanding open unresolved discussion about this: https://github.com/zephyrproject-rtos/zephyr/pull/25590#pullrequestreview-518296792

Pinging @anangl about this

Reply:
I was following this example here which also uses differential channels.
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/adc/ti%2Clmp90xxx-base.yaml

Wouldn't this have the same issue?

Reply:
Looks like yes, it would have the same issue. That was added in https://github.com/zephyrproject-rtos/zephyr/pull/20381. If I had reviewed that PR and been aware of this issue at the time I would have said the same thing.

cc @henrikbrixandersen since it's for a driver he added

Reply:
What do you mean differential channels aren't supported? I use differential channels in the LMP90xxx.

Reply:
That must be using a nonstandard binding then -- see the investigation in https://github.com/zephyrproject-rtos/zephyr/pull/25590#pullrequestreview-518296792 I linked to above

Reply:
done. changed to 1